mantidimaging/gui/windows/spectrum_viewer/view.py

```diff
@@ -98,7 +107,55 @@ def __init__(self, main_window: 'MainWindowView'):
         self.tableView.setSelectionMode(QAbstractItemView.SingleSelection)
         self.tableView.setAlternatingRowColors(True)
 
+        # Roi Prop table
+        self.roi_table_properties = ["Top", "Bottom", "Left", "Right"]
+        self.roi_table_properties_secondary = ["Width", "Height"]
+        self.roiPropertiesTableWidget.setColumnCount(3)
+        self.roiPropertiesTableWidget.setRowCount(3)
+        self.roiPropertiesTableWidget.setColumnWidth(0, 80)
+        self.roiPropertiesTableWidget.setColumnWidth(1, 50)
+        self.roiPropertiesTableWidget.setColumnWidth(2, 50)
+
+        self.roiPropertiesSpinBoxes = {}
+        self.roiPropertiesLabels = {}
+        for prop in self.roi_table_properties:
+            spin_box = QSpinBox()
+            if prop == "Top" or prop == "Bottom":
+                spin_box.setMaximum(self.spectrum_widget.image.image_data.shape[0])
+            if prop == "Left" or prop == "Right":
+                spin_box.setMaximum(self.spectrum_widget.image.image_data.shape[1])
+            spin_box.valueChanged.connect(self.adjust_roi)
+            self.roiPropertiesSpinBoxes[prop] = spin_box
+        for prop in self.roi_table_properties_secondary:
+            label = QLabel()
+            self.roiPropertiesLabels[prop] = label
+
+        self.roiPropertiesTableWidget.horizontalHeader().hide()
+        self.roiPropertiesTableWidget.verticalHeader().hide()
+        self.roiPropertiesTableWidget.setShowGrid(False)
+
+        roiPropertiesTableText = ["x1, x2", "y1, y2", "Width, Height"]
+        self.roiPropertiesTableTextDict = {}
+        for text in roiPropertiesTableText:
+            item = QTableWidgetItem(text)
+            item.setFlags(Qt.ItemIsSelectable)
+            self.roiPropertiesTableTextDict[text] = item
+
+        self.roiPropertiesTableWidget.setItem(0, 0, self.roiPropertiesTableTextDict["x1, x2"])
+        self.roiPropertiesTableWidget.setCellWidget(0, 1, self.roiPropertiesSpinBoxes["Left"])
+        self.roiPropertiesTableWidget.setCellWidget(0, 2, self.roiPropertiesSpinBoxes["Right"])
+        self.roiPropertiesTableWidget.setItem(1, 0, self.roiPropertiesTableTextDict["y1, y2"])
+        self.roiPropertiesTableWidget.setCellWidget(1, 1, self.roiPropertiesSpinBoxes["Top"])
+        self.roiPropertiesTableWidget.setCellWidget(1, 2, self.roiPropertiesSpinBoxes["Bottom"])
+        self.roiPropertiesTableWidget.setItem(2, 0, self.roiPropertiesTableTextDict["Width, Height"])
```

Comment:
This currently does not fit the default width. Maybe reword to "Size"
![image](https://github.com/mantidproject/mantidimaging/assets/32413847/92872813-ff03-4809-9e99-212dd574c9c7)